File: iSpindel.Database/BeerCharacteristics.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace iSpindel.Database {
    public class BeerCharacteristics {
        public int Id {get; set;}

        [ForeignKey("DataSeriesId")]
        public virtual DataSeries DataSeries { get; set; }
        public int DataSeriesId {get; set;}
        public double? EVG { get; set; }
        public int? Bitterness { get; set; }
        public double? ColorScale { get; set; }
        public double? BrewhouseEfficency { get; set; }
        public double? AmountOfWort { get; set; }
        public double? TargetCarbonation { get; set; }
        public double? AddedSugar { get; set; }
        public double? AdjustedAlcoholLevel { get; set; }
        #nullable enable
        public string? YeastType { get; set; }
        public string? Notes {get; set;}
        public string? BeerStyle {get; set;}
        #nullable restore

    }
}

File: iSpindel.Frontend/iSpindel.Frontend/src/classes/Data/BeerCharacteristics.ts
import { IBeerCharacteristics } from "./IBeerCharacteristics";

export class BeerCharacteristics implements IBeerCharacteristics {
    evg: number;
    bitterness: number;
    colorScale: number;
    brewhouseEfficency: number;
    amountOfWort: number;
    targetCarbonation: number;
    addedSugar: number;
    adjustedAlcoholLevel: number;
    yeastType: string;
    notes: string;
    beerStyle: string;

    public getEvg() : string {
        return this.evg == null ? "" : this.evg.toString() + " %";
    }

    public getBitterness() : string {
        return this.bitterness == null ? "" : this.bitterness.toString() + " IBU";
    }
    
    public getColorScale(): string {
        return this.colorScale == null ? "" : this.colorScale.toString() + " EBC";
        
    }

    public getBrewhouseEfficency() : string {
        return this.brewhouseEfficency == null ? "" : this.brewhouseEfficency.toString() + " %";
    }

    public getAmountOfWort() : string {
        return this.amountOfWort == null ? "" : this.amountOfWort.toString() + " L";
    }

    public getAmountOfWortUnit(): string{
        return "L";
    }

    public getTargetCarbonation() : string {
        return this.targetCarbonation == null ? "" : this.targetCarbonation.toString() + " g/L";
    }

    public getAddedSugar() : string {
        return this.addedSugar == null ? "" : this.addedSugar.toString() + " g/L";
    }

    public getAdjustedAlcoholLevel() : string {
        return this.adjustedAlcoholLevel == null ? "" : this.adjustedAlcoholLevel.toString() + " g/L";
    }

}